### PR TITLE
Remove challenge questions related integration tests

### PR DESCRIPTION
**Pull request by asha15**
$subject

Related issue: https://github.com/wso2/product-is/issues/17749, https://github.com/wso2/product-is/issues/16629

### CODE REVIEW

File: modules/integration/tests-common/admin-clients/pom.xml [hunk 1]
```diff
@@ -272,6 +268,13 @@
             <groupId>org.wso2.carbon.registry</groupId>
             <artifactId>org.wso2.carbon.registry.properties.stub</artifactId>
         </dependency>
+        <!-- Using the previous stub version for compilation due to deprecation of the challenge question management feature in IS 7.0.0 and onwards -->
+        <dependency>
+            <groupId>org.wso2.carbon.identity.governance</groupId>
+            <artifactId>org.wso2.carbon.identity.recovery.stub</artifactId>
+            <version>1.8.87</version>
+            <scope>compile</scope>
+        </dependency>
```
Comment (chamathns) [inline modules/integration/tests-common/admin-clients/pom.xml:277]: this would be temporary yes?

Comment (asha15) [reply modules/integration/tests-common/admin-clients/pom.xml:277]: yes it's. After hosting the challenge questions as a connector we need to improve the test cases to download it and run the test on top of it. Afterwards we can remove this.

Comment (chamathns) [reply modules/integration/tests-common/admin-clients/pom.xml:277]: shall we add a comment here explaining that?

Comment (asha15) [reply modules/integration/tests-common/admin-clients/pom.xml:282]: fixed with https://github.com/wso2/product-is/pull/17772/commits/9d22f845ed9a6e826e9ee34f70d395b7e1a67edf

**Review by chamathns — APPROVED** [review]

**Pull request merged**